styx-scheduler-service/src/test/java/com/spotify/styx/StyxSchedulerServiceFixture.java

```diff
@@ -114,26 +108,20 @@
   private StyxScheduler styxScheduler;
   private ServiceHelper serviceHelper;
 
-  @BeforeClass
-  public static void setUpClass() throws Exception {
-    // TODO: the datastore emulator behavior wrt conflicts etc differs from the real datastore
+  public StyxSchedulerServiceFixture() {
     localDatastore = LocalDatastoreHelper.create(1.0); // 100% global consistency
-    localDatastore.start();
-  }
-
-  @AfterClass
-  public static void tearDownClass() throws Exception {
-    if (localDatastore != null) {
-      try {
-        localDatastore.stop(org.threeten.bp.Duration.ofSeconds(30));
-      } catch (Throwable e) {
-        e.printStackTrace();
-      }
+    try {
+      localDatastore.start();
```

Comment:
This could be very slow stop/start emulator for each test case. If that is to make it more reliable, the code in constructor I think better to to `setUp`.

Reply:
This makes circleci very sad. :D

Reply:
Yeah, I'm actively experimenting here a bit with how to make the tests work